src/commands/importExportEntities/entities/utils/richText.ts

```diff
@@ -0,0 +1,27 @@
+import { ImportContext } from "../../entityDefinition.js";
+
+export const replaceRichTextReferences = (richText: string, context: ImportContext): string =>
+  richText
+    .replaceAll(assetRegex, (_, assetId) => {
+      const newAssetId = context.assetIdsByOldIds.get(assetId);
+      if (!newAssetId) {
+       console.warn(`Found asset id "${assetId}" in rich text that doesn't exist.`);
+      }
+      
+      return `${assetAtributeName}="${newAssetId}"`;
+    })
+    .replaceAll(itemRegex, (_, itemId) => {
```

Comment:
so Item Id is obtained from the regex group? maybe add comment to clear that up :)